parts/k8s/addons/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -75,7 +75,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
         - name: azure-npm
-          image: containernetworking/azure-npm:v1.0.9
+          image: containernetworking/azure-npm:v1.0.13

```

Comment:
why is this version changing and how is this related to moby?

Reply:
hmm, rebase errata